src/utils/validationService.ts

```diff
@@ -0,0 +1,22 @@
+import CertificateNumberError from '../errors/CertificateNumberError';
+import VinError from '../errors/VinError';
+import CertificateDetails from '../interfaces/CertificateDetails';
+
+const certRegex = /^[a-z-A-Z]\d{2}[a-z-A-Z]\d{5}$/;
+// The below is based on the ISO 3779:2009 standard from
+// https://cdn.standards.iteh.ai/samples/52200/7d8a69aee84c4ad28231053f49f4966e/ISO-3779-2009.pdf
+// It simplifies the sections of the VIN (WMI, VDS, VIS) as, apart from the last four digits, they're all alphanumeric
+// excluding the letters I, O and Q
+const vinRegex = /^[a-hA-Hj-nJ-NpPr-zR-Z\d]{13}\d{4}$/;
+
+export default (event: CertificateDetails): boolean => {
```

Comment:
So just a general comment, feel free to ignore, but I feel like TS "services" with a small number of exported functions would usually be named in terms of those functions, e.g. `validate-certificate-details.ts`, as opposed to `validationService.ts`, which to me implies something OOP like a class

Reply:
Yeah, I know where you're coming from. Just wondering if `validateVinAndCertificateNumber` is a bit long for a filename